pkg/kubectl/cmd/clusterinfo_dump.go

```diff
@@ -46,10 +45,11 @@ type ClusterInfoDumpOptions struct {
 	AllNamespaces bool
 	Namespaces    []string
 
-	timeout       time.Duration
-	clientset     internalclientset.Interface
-	namespace     string
-	logsForObject func(object, options runtime.Object, timeout time.Duration) (*restclient.Request, error)
+	timeout          time.Duration
+	clientset        internalclientset.Interface
+	namespace        string
+	restClientGetter polymorphichelpers.RESTClientGetter
+	logsForObject    polymorphichelpers.LogsForObjectFunc
```

Comment:
I think we discussed about it, so since you're touching would you mind making these public? You'll need to rebase, either way :wink: